app/src/androidTest/java/com/github/sdp/tarjetakuna/ui/HomeFragmentTest.kt

```diff
@@ -29,9 +31,24 @@ class HomeFragmentTest {
 
         activityRule.scenario.onActivity { activity ->
             val navController = findNavController(activity, R.id.nav_host_fragment_content_drawer)
-            assertThat(navController.currentDestination?.id, equalTo(R.id.nav_authentication_button))
+            assertThat(
+                navController.currentDestination?.id,
+                equalTo(R.id.nav_authentication_button)
+            )
         }
 
     }
 
+//    @Test
```

Comment:
Why this test is in comment?

Reply:
Because I forgot to uncomment it, thanks for the reminder